### PR TITLE
BD-2383: Hidden doc for new email engagement dashboard graphs

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-2383**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_hidden/private_betas/email_engagement.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Email engagement dashboard
+
+To help track and better understand your email engagement metrics, you can reference the **Engagement by Mailbox Provider** and **Time of Engagement** graphs in the email engagement dashboard. 
+
+{% alert important %}
+These graphs for the email engagement dashboard are currently in early access. Contact your Braze account manager if you're interested in participating in the early access.
+{% endalert %}
+
+Engagement data may include machine opens and machine clicks, and these metrics are not separately or excluded from the reports. However, Braze plans to provide such support if and when this feature generally available.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/email_engagement.md:17]:
```suggestion
Engagement data may include machine opens and machine clicks, and these metrics are not separated or excluded from the reports. However, Braze plans to provide such support if and when this feature is generally available.
```

File: _docs/_hidden/private_betas/email_engagement.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### Engagement by Mailbox Provider
+
+The **Engagement by Mailbox Provider** graph shows the number of clicks and opens by mailbox provider. You can click a mailbox provider and drilldown into specific receiving domains. For example, if you have Microsoft as a mailbox provider, you would be able to further view details for "outlook.com" or "live.com".
+
+### Time of Engagement
+
+With the **Time of Engagement** graph, you can view when users are engaging with their emails. This can help answer questions such as which weekday or what time sees the highest engagement from your customers. Note that these times are based on your workspace's timezone.
+
+For more information on analytics for your emails, check out [Email reporting]({{site.baseurl}}/user_guide/message_building_by_channel/email/reporting_and_analytics/email_reporting/)
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/email_engagement.md:29]:
```suggestion
For more information on analytics for your emails, check out [Email reporting]({{site.baseurl}}/user_guide/message_building_by_channel/email/reporting_and_analytics/email_reporting/).
```
Added period

**Review by rachel-feinberg — APPROVED** [review]
Lgtm, just left a suggestion to add a period!

**Comment by lydia-xie** [comment]
@cla-bot check

**Pull request merged**